includes/Feature.php

```diff
@@ -45,7 +45,7 @@ final public function __construct( $options ) {
 		$this->options = $options;
 
 		// check if state already saved to options
-		$this->setValue( $this->options->get( $this->name ) );
+		$this->setValue( $this->options->get( $this->name ) ? $this->options->get( $this->name ) : $this->value );
```

Comment:
```suggestion
		$this->setValue( $this->options->get( $this->name, $this->value ) );
```

See https://github.com/wp-forge/wp-options/blob/master/includes/Options.php#L74